### PR TITLE
UI: wrong keypool default value displayed

### PR DESCRIPTION
**Pull request by crowning-**
Reference: https://dashtalk.org/threads/v12-release.5888/page-58#post-69159 (first chapter)

Created DEFAULT_KEYPOOL_SIZE constant to ease maintenance.

### CODE REVIEW

File: src/rpcwallet.cpp [hunk 1]
```diff
@@ -1685,7 +1685,7 @@ Value keypoolrefill(const Array& params, bool fHelp)
             "\nFills the keypool."
             + HelpRequiringPassphrase() + "\n"
             "\nArguments\n"
-            "1. newsize     (numeric, optional, default=100) The new keypool size\n"
+            "1. newsize     (numeric, optional, default=1000) The new keypool size\n"
```
Comment (UdjinM6) [inline src/rpcwallet.cpp]: How about `" + itostr(DEFAULT_KEYPOOL_SIZE) + "` instead of `1000` here too?

**Comment by crowning-** [comment]
Will do after work.

**Comment by crowning-** [comment]
Added and ready to merge. Thanks @UdjinM6 :thumbsup:

**Comment by UdjinM6** [comment]
Looks good :+1:

**Pull request merged**